templates/CRM/Member/Form/MembershipRenewal.tpl

```diff
@@ -8,6 +8,9 @@
  +--------------------------------------------------------------------+
 *}
 {* this template is used for renewing memberships for a contact  *}
+{if $priceSetOnly}
```

Comment:
If you are doing this to load a new set of values onto the form (ie an overload) we are working to put them in separate form / template classes with their own paths rather than having templates that are basically big IF- elses

Reply:
@eileenmcnaughton Any examples of forms that did that recently? (I'd also like to switch this form to `snippet=json`, because `snippet=4` is a pain for extensions who might add custom JS/vars)